modules/installing-ove-console-final.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-booting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-console-final_{context}"]
+= Completing cluster configuration and initiating the installation
+
+Before you can finally initiate the cluster installation, you must verify host details, configure cluster networking details, and download the default cluster credentials.
+
+.Prerequisites
+
+* You have booted all of the hosts that will comprise your cluster and configured them with the correct rendezvous node IP address.
++
+[IMPORTANT]
+====
+You can add as many hosts as you want to your cluster.
+However, at this stage, you must at least have enough available hosts to match the value you specified in the *Number of control plane nodes* field of the installation console's *Cluster details* page.
```

Comment:
This note was my attempt at saying "You can add more nodes than the number of control planes you specified earlier, but you can't proceed further if you didn't boot enough machines to make up the control plane". 

Let me know if it's inaccurate or unnecessary.